backend/api/v1/auth_service.go

```diff
@@ -437,37 +438,66 @@ func (s *AuthService) getUserWithLoginRequestOfBytebase(ctx context.Context, req
 func (s *AuthService) getUserWithLoginRequestOfIdentityProvider(ctx context.Context, request *v1pb.LoginRequest) (*store.UserMessage, error) {
 	identityProviderID, err := getIdentityProviderID(request.IdpName)
 	if err != nil {
-		return nil, status.Errorf(codes.InvalidArgument, err.Error())
+		return nil, status.Error(codes.InvalidArgument, err.Error())
 	}
 
 	identityProvider, err := s.store.GetIdentityProvider(ctx, &store.FindIdentityProviderMessage{
 		ResourceID: &identityProviderID,
 	})
 	if err != nil {
-		return nil, status.Errorf(codes.Internal, "failed to get identity provider")
+		return nil, status.Errorf(codes.Internal, "failed to get identity provider: %v", err)
 	}
 	if identityProvider == nil {
-		return nil, status.Errorf(codes.NotFound, "identity provider not found")
```

Comment:
Better to use Errorf consistently

Reply:
`Errorf` implies formatting, which IMO is a misuse. Similar to `fmt.Print` vs `fmt.Printf`, `errors.Wrap` vs `errors.Wrapf`.

Reply:
Yup there is no reason to use Print().

Reply:
I disagree but this is non-blocking for me, updated in https://github.com/bytebase/bytebase/pull/4462/commits/3f6f4689c0331c27d2e33b1faaac973740dfbb07